src/components/shared/partials/FormInputs.jsx

```diff
@@ -0,0 +1,70 @@
+export function Inputs({ className, formData, onChange, validated, checkEmpty, errormsg, required }) {
+	const { type, minLength, maxLength, regex, id, placeholder, value } = formData;
+
+	const validateInput = event => {
+		const value = event.target.value;
+		if (regex && value) {
+			const isValid = value.match(regex);
+			const isEmpty = !required || value === '';
+
+			if (!isValid) {
+				console.log(`Invalid input for ${id}`);
+				event.target.style.border = '1px solid #b91c1c';
+
+				validated(prev => ({
+					...prev,
+					[id]: false,
+				}));
+				errormsg(prevMsg => prevMsg + `\nInvalid input for ${id}`);
```

Comment:
Why not make thes reusable functions.

Reply:
wrote the function https://github.com/dscnitrourkela/project-udon/pull/11/commits/0f45d926f61a61c014e1253bb5307c9bc3f9c98d